Packages/io.chainsafe.web3-unity/Runtime/Scripts/EVM/Token/Erc721.cs

```diff
@@ -36,13 +36,28 @@ public static async Task<int> BalanceOf(Web3 web3, string contractAddress, strin
         /// <param name="tokenId"></param>
         /// <returns></returns>
         public static async Task<string> OwnerOf(Web3 web3, string contractAddress, string tokenId)
+        {
+            return await OwnerOf(web3, contractAddress, new object[] { tokenId, });
+        }
+        
+        /// <summary>
+        /// Owner Of ERC721 Token
+        /// </summary>
+        /// <param name="web3"></param>
+        /// <param name="contractAddress"></param>
+        /// <param name="tokenId"></param>
+        /// <returns></returns>
+        public static async Task<string> OwnerOf(Web3 web3, string contractAddress, int tokenId)
```

Comment:
Are we 100% sure that the tokenId will never exceed ints maxvalue? Should it be biginteger maybe instead?

Reply:
I'm not sure but sounds valid, thanks Nikola will update

Reply:
So an example of when this could become an issue is, as example, the original ChainSafe ERC1155 NFT contract.

We encode the IPFS media identifier (A hash to look up a file on distributed storage), we encoded the media hash, into it's tokenId.

This means the tokenId = the unique media hash. The issue is, as a number, it is extremely big, and would likely overflow most `Int` types.

So personally, I would prefer to not even entertain the idea that we would use `Int` at all except for smaller Solitity types, like `Uint128` or below. But thats a whole lot of edge case checking


For reference, the max `Uint256` as an integer
```
// uint256 MAX_INT = uint256(-1)
uint256 MAX_INT = 115792089237316195423570985008687907853269984665640564039457584007913129639935
```

Reply:
BigInteger for sure!

Reply:
Updated